src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -541,31 +541,9 @@ class Manager : public DoutPrefixProvider {
     ldpp_dout(this, 20) << "INFO: queue: " << topic_name << " added to queue list"  << dendl;
     return 0;
   }
-  
+
   int remove_persistent_topic(const std::string& topic_name, optional_yield y) {
```

Comment:
you actually dont need this function. you can just call the standalone function in line 620, passing `s_manager->...` calues

Reply:
Done!